docs/contribute/release_checklist.md

```diff
@@ -4,40 +4,155 @@ title: Release checklist
+    index 895fa9d69f0..f62d9ea6efe 100644
+    --- a/docs/changelog.md
+    +++ b/docs/changelog.md
+    @@ -4,20 +4,26 @@ title: Changelog
+    permalink: /changelog/
+    ---
+
+    -# 98.0.0 (In Development)
+    -* [Commits](https://github.com/mozilla-mobile/android-components/compare/v97.0.0...main)
+    -* [Milestone](https://github.com/mozilla-mobile/android-components/milestone/145?closed=1)
+    +# 99.0.0 (In Development)
+    +* [Commits](https://github.com/mozilla-mobile/android-components/compare/v98.0.0...main)
+    +* [Milestone](https://github.com/mozilla-mobile/android-components/milestone/146?closed=1)
+    * [Dependencies](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Dependencies.kt)
+    * [Gecko](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt)
+    * [Configuration](https://github.com/mozilla-mobile/android-components/blob/main/.config.yml)
+
+    +# 98.0.0
+    +* [Commits](https://github.com/mozilla-mobile/android-components/compare/v97.0.0...v98.0.0)
+    +* [Milestone](https://github.com/mozilla-mobile/android-components/milestone/145?closed=1)
+    +* [Dependencies](https://github.com/mozilla-mobile/android-components/v98.0.0/main/buildSrc/src/main/java/Dependencies.kt)
+    +* [Gecko](https://github.com/mozilla-mobile/android-components/v98.0.0/main/buildSrc/src/main/java/Gecko.kt)
+    +* [Configuration](https://github.com/mozilla-mobile/android-components/v98.0.0/main/.config.yml)
+    +
+    ```
+
+5. Create a commit named `Update to GeckoView [nightly_version].0` that bumps the GeckoView `version` in [Gecko.kt](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt) to the latest `[nightly_version]` build when available in [maven/geckoview-nightly](https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/geckoview-nightly/).
+
+    ```diff
+    diff --git a/buildSrc/src/main/java/Gecko.kt b/buildSrc/src/main/java/Gecko.kt
+    index 991dd702bb5..6f6c4471a26 100644
+    --- a/buildSrc/src/main/java/Gecko.kt
+    +++ b/buildSrc/src/main/java/Gecko.kt
+    @@ -9,7 +9,7 @@ object Gecko {
+        /**
+          * GeckoView Version.
+          */
+    -    const val version = "98.0.20220207065816"
+    +    const val version = "99.0.20220208070047"
+
+        /**
+          * GeckoView channel
+    ```
+
+6. Create a pull request with these 3 commits and land. If this is landed after the scheduled [cron task](https://github.com/mozilla-mobile/android-components/blob/main/.cron.yml#L13) build that will build and bump AC automatically, trigger a manual AC build through the [hook](https://firefox-ci-tc.services.mozilla.com/hooks/project-releng/cron-task-mozilla-mobile-android-components%2Fnightly). If a manual AC build was needed, trigger the [hook](https://firefox-ci-tc.services.mozilla.com/hooks/project-releng/cron-task-mozilla-mobile-fenix%2Fbump-android-components) to bump
```

Comment:
to bump..... A-C in Fenix. (sentence not finished.)

As discussed, I think we should just land these before the regular scheduled nightlies going forward. Should be easy enough to do, but good to document this in case we miss it.

Reply:
Fixed.